src/main/java/no/unit/nva/model/contexttypes/Periodical.java

```diff
@@ -0,0 +1,8 @@
+package no.unit.nva.model.contexttypes;
+
+/**
+ * Marker pattern parent interface for Journal and UnconfirmedJournal.
```

Comment:
Although I understand the "parent interface", I do not understand the "pattern parent interface" and the "marker pattern parent interface"